hikari/impl/bot.py

```diff
@@ -125,6 +125,32 @@ async def murder(future: asyncio.Future[typing.Any]) -> None:
     # Closed loops cannot be re-used so it should also be un-set.
     asyncio.set_event_loop(None)
 
+
+@staticmethod
```

Comment:
This is no longer needed. Also, TIL that you can have this outside of a class without it erroring, interesting